### DISCUSSION TITLE
WRF Installation Sonoma 14.6.1 (on Macbook pro M2), did anybody test it

### DISCUSSION THREAD

**Discussion by srivastava1978** [HathewayWill/WRF-MOSIT#44] [Q&A]
Dear Sir,
I am trying to install WRF using WRF-MOSIT script, but till now unsuccessful, some issues which I could see is 
1. it is searching for Python3.8, which is no more available
2. stuck with message "libtool: link: waiting for /Volume/....../libmiutil.a.lock to be removed"
3. dateutil issue, which is perhaps resolved after making 38 to 31 in script and installing it though vm.
4. It says that it needs 320 GB of space, I have mounted my SSD and made it HOME using export command. 
As for now, it is stuck with "libtool: link: waiting for /Volume/....../libmiutil.a.lock to be removed"
Is there any updated script, I am tired, I tried all ways. Please help!
Best regards
Shailendra

**Comment by HathewayWill** [comment]
@srivastava1978 

Good morning,
First of all I am sorry that you are having issues.

Could you provide me with some information.

1. Which configuration options did you choose
2. What operating system are you using?
3. Did you create a log file for the script like the website shows?  If so can you attach it.
4. What type of computer are you using? Laptop? Desktop? HPC?

**Comment by srivastava1978** [comment]
Thank you so much for your quick response.

I will answer one by one:
1. For normal installation, I used Serial + gcc-gfortran, and for
WRF-MOSIT, it did not ask for any such thing.
2. Mac OS Sonoma 14.6.1 on macbook pro M2
3. log file is attached for your reference.
4. Laptop

Please let me know, I would be very grateful for your help.
Regards
Shailendra

On Fri, Nov 8, 2024 at 3:40 PM HathewayWill ***@***.***>
wrote:

> @srivastava1978 <https://github.com/srivastava1978>
>
> Good morning,
> First of all I am sorry that you are having issues.
>
> Could you provide me with some information.
>
>    1. Which configuration options did you choose
>    2. What operating system are you using?
>    3. Did you create a log file for the script like the website shows? If
>    so can you attach it.
>    4. What type of computer are you using? Laptop? Desktop? HPC?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/HathewayWill/WRF-MOSIT/discussions/44#discussioncomment-11187437>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A7WVALZYLEBZAOWRKJBU3FLZ7SEYFAVCNFSM6AAAAABRNDSCMWVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCMJYG42DGNY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Warm regards.

Shailendra S. Srivastava

Scientist-SF (Retd.), SAC, Ahmedabad
[image: envelope]

***@***.***
[image: phone]

(+91) 966-49-39-208

Let us consciously choose to respond with peace, calmness, and happiness
today! Be kind whenever possible. It is always possible. ~ Dalai Lama

**Reply by HathewayWill** [reply]
didn't see the attached log file, can you zip it an attach it

**Comment by HathewayWill** [comment]
try installing it now, fixed a few issues in the scripts.  Delete the folders that were already downloaded and reclone them down again

**Comment by srivastava1978** [comment]
The problem persist with libtool: link: Waiting for /Volumes/SSS_SSD/WRF/Downloads/mpich-4.2.3/src/pmi/./.libs/libpmiutil.a.lock to be removed
libtool: link: Waiting for /Volumes/SSS_SSD/WRF/Downloads/mpich-4.2.3/src/pmi/./.libs/libpmiutil.a.lock to be removed
.
.repeated same message....
.
.
I tried to delete that file, but it does not exist....
logfile attached
[WRF_MOSIT.log](https://github.com/user-attachments/files/17685995/WRF_MOSIT.log)

**Reply by HathewayWill** [reply]
Is this being build on an internal or external hard drive

**Reply by HathewayWill** [reply]
> The problem persist with libtool: link: Waiting for /Volumes/SSS_SSD/WRF/Downloads/mpich-4.2.3/src/pmi/./.libs/libpmiutil.a.lock to be removed libtool: link: Waiting for /Volumes/SSS_SSD/WRF/Downloads/mpich-4.2.3/src/pmi/./.libs/libpmiutil.a.lock to be removed . .repeated same message.... . . I tried to delete that file, but it does not exist.... logfile attached [WRF_MOSIT.log](https://github.com/user-attachments/files/17685995/WRF_MOSIT.log)

this error usually is due to permission issues, try running the script again but do chmod 777

**Comment by srivastava1978** [comment]
[WRF_MOSIT.log](https://github.com/user-attachments/files/17688773/WRF_MOSIT.log)

Please check the log, I am sending screenshots of the same of your email.
Please help!

**Comment by HathewayWill** [comment]
I'll try to get to it next week.  It's the weekend and I have family
visiting
